backend/app/api/v1/setting.go

```diff
@@ -354,6 +371,7 @@ func (b *BaseApi) GenerateApiKey(c *gin.Context) {
 // @Param request body dto.ApiInterfaceConfig true "request"
 // @Success 200
 // @Security ApiKeyAuth
+// @Security Timestamp
 // @Router /settings/api/config/update [post]
 // @x-panel-log {"bodyKeys":["ipWhiteList"],"paramKeys":[],"BeforeFunctions":[],"formatZH":"更新 API 接口配置 => IP 白名单: [ipWhiteList]","formatEN":"update api config => IP White List: [ipWhiteList]"}
 func (b *BaseApi) UpdateApiConfig(c *gin.Context) {
```

Comment:
There is no specific code provided to review. Please provide the relevant code, and I can assist you with any discrepancies found.